vector/src/main/java/im/vector/app/features/location/Config.kt

```diff
@@ -22,5 +22,5 @@ const val DEFAULT_PIN_ID = "DEFAULT_PIN_ID"
 
 const val INITIAL_MAP_ZOOM_IN_PREVIEW = 15.0
 const val INITIAL_MAP_ZOOM_IN_TIMELINE = 17.0
-const val MIN_TIME_TO_UPDATE_LOCATION_MILLIS = 2 * 1_000L // every 2 seconds
+const val MIN_TIME_TO_UPDATE_LOCATION_MILLIS = 5 * 1_000L // every 5 seconds
```

Comment:
we could use 5.seconds.inWholeMilliseconds here but not sure we want to use this everywhere

Reply:
I agree it is a better syntax. The drawback is that it can no more be a `const`. I updated the code, see: [49ad37a](https://github.com/vector-im/element-android/pull/6654/commits/49ad37a7f69b9d59863c6afb6a9591ee93d37824)